src/views/Signup.js

```diff
@@ -292,7 +292,9 @@ export default class Signup extends Component {
                   </option>
 
                   {roleOptions.map((option) => (
-                    <option value={option.value}>{option.label}</option>
+                    <option key={option.value} value={option.value}>
```

Comment:
For insufficiently strong passwords, I can see the error state come through but the create action still works instead of keeping me on the modal. For mismatching passwords, I am correctly kept on the modal and can see the error.

Reply:
![image](https://user-images.githubusercontent.com/13375042/141495689-8e78f2b2-344f-4619-9f3e-fa61a2d53d66.png)